lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view_web.dart

```diff
@@ -107,7 +108,7 @@ class MailboxDashBoardView extends BaseMailboxDashBoardView {
                     children: [
                       SizedBox(child: MailboxView(), width: responsiveUtils.defaultSizeMenu),
                       Expanded(child: Column(children: [
-                        _buildSpamReportBoxWebWidget(context),
+                        const SpamReportBannerWebWidget(),
                         _buildEmptyTrashButton(context),
                         const QuotasWarningBannerWidget(
                           margin: EdgeInsets.only(right: 16, top: 8),
```

Comment:
make it `const EdgeInsets.only(right: 16, top: 8)`

Reply:
const QuotasWarningBannerWidget(
                          margin: EdgeInsets.only(right: 16, top: 8),  
                          
:)))